---
layout: archive
title: "Publications"
permalink: /publications/
author_profile: true
---

Below is a list of selected publications and preprints. For a full list, please visit my [Google Scholar page](https://scholar.google.be/citations?user=3PHGkxkAAAAJ&hl=en). My work has mostly focussed on differential expression analysis of high-throughput sequencing datasets.

\* denotes shared first authorship and &dagger; denotes shared corresponding authors.

# (Applied) statistical methodology

- Gillis J., Vitting-Seerup K., **Van den Berge K.**&dagger;, Clement L.&dagger;. [satuRn: Scalable Analysis of differential Transcript Usage for bulk and single-cell RNA-sequencing applications](https://f1000research.com/articles/10-374/v1). *F1000Research* (in open review).
- Nguyen K., **Van den Berge K.**, Chiogna M., Risso D. (2020). [Structure learning for zero-inflated counts, with an application to single-cell RNA sequencing data](https://arxiv.org/abs/2011.12044). *arXiv*
- Li T., Lei L., Bhattacharyya S., **Van den Berge K.**, Sarkar P., Bickel P.J., Levina E. (2020). [Hierarchical community detection by recursive partitioning](https://www.tandfonline.com/doi/full/10.1080/01621459.2020.1833888). *Journal of the American Statistical Association* (in print).
- **Van den Berge K.**, Roux de B'ezieux H., Street K., Saelens W., Cannoodt R., Saeys Y., Dudoit S.&dagger;, Clement L.&dagger; (2020). [Trajectory-based differential expression analysis for single-cell sequencing data](https://www.nature.com/articles/s41467-020-14766-3). *Nature Communications* 11:1201.
- **Van den Berge K.\***, Hembach K.M.\*, Soneson C.\*, Tiberi S.\*, Clement L.&dagger;, Love M.I.&dagger;, Patro R.&dagger;, Robinson M.D.&dagger; (2019) [RNA Sequencing Data: Hitchhiker's Guide to Expression Analysis](https://www.annualreviews.org/doi/10.1146/annurev-biodatasci-072018-021255). *Annual Review of Biomedical Data Science* 2:139-173.
- **Van den Berge K.\***, Perraudeau F.\*, Soneson C., Love M.I., Risso D., Vert J.P., Robinson M.D., Dudoit S., Clement L. (2018). [Observation weights unlock bulk RNA-seq tools for zero inflation and single-cell applications](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1406-4). *Genome Biology* 19:24.
- **Van den Berge K.\***, Soneson C., Robinson M.D., Clement L. (2017). [stageR: a general stage-wise method for controlling the gene-level false discovery rate in differential expression and differential transcript usage](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1277-0). *Genome Biology* 18:151.
 
# Collaborative papers
 
 - [Mating type specific transcriptomic response to sex inducing pheromone in the pennate diatom Seminavis robusta](https://www.nature.com/articles/s41396-020-00797-7)
 - [Non-neuronal expression of SARS-CoV-2 entry genes in the olfactory system suggests mechanisms underlying COVID-19-associated anosmia](https://advances.sciencemag.org/content/6/31/eabc5801.abstract)
 - [Neurogenomic profiling reveals distinct gene expression profiles between brain parts that are consistent in Ophthalmotilapia cichlids](https://www.frontiersin.org/articles/10.3389/fnins.2018.00136/full?report=reader)
 - [A sex-inducing pheromone triggers cell cycle arrest and mate attraction in the diatom Seminavis robusta](https://www.nature.com/articles/srep19252)
 
